stix_shifter_modules/alertflex/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,48 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "src_ip": ["a.srcIp"],
+      "dst_ip": ["a.dstIp"],
+      "ip": ["a.dstIp", "a.srcIp"]
```

Comment:
None of these are properties of the `ipv4-addr` object. You should have `"value": ["a.dstIp", "a.srcIp"]`; the source and destination ip can be searched from network-traffic object.